### DISCUSSION TITLE
GLB viewer doesn't work with the latest version

### DISCUSSION THREAD

**Discussion by shinthom** [google/model-viewer#3623] [Q&A]
### Description

I've been using `model-viewer` to make GLB viewer, but the GLB file doesn't work with the latest version.

The GLB file is exported from Unreal Engine.

To resolve this issue, we continued to downgrade the `model-viewer` package until the GLB viewer worked.

Finally, the GLB viewer worked in version 1.10.1.

```json
"@google/model-viewer": "1.10.1",
"lit-element": "2.5.1",
"lit-html": "1.4.1",
"three": "0.136.0",

-"@google/model-viewer": "^1.12.0",
```

I'd like to use the latest version, so could you tell me the information related to this issue?

#### Live Demo

Your online viewer seems to be using the latest version. I tested as below using two online viewers (including your viewer).

- https://modelviewer.dev/editor/ (not working)
- https://gltf-viewer.donmccurdy.com/ (working)

![112](https://user-images.githubusercontent.com/102012918/179144413-efc3c9fd-f5aa-4b09-9f67-0df63e055a19.gif)

### Browser Affected

Chrome, version: 103.0.5060.114

### OS

MacOS, version: 12.2.1

**Comment by elalish** [comment]
Can you please include your non-working file so we can repro? There are some open issues regarding certain kinds of animation transforms and mesh quantization that might be related. If that causes the bounding box computation to be off, then the camera points wrong and you see nothing.

**Comment by shinthom** [comment]
[not-working.glb.zip](https://github.com/google/model-viewer/files/9124774/not-working.glb.zip)

Thanks for replying. I can't upload the glb file to github, so I'm uploading it as a zip file.

**Comment by elalish** [comment]
Okay, so I got it to work (sort of). If you zoom in, you can just see it. If you set the camera to 5% radius, it becomes more visible; it looks correct, but the animation doesn't play. The glTF validator also has *a lot* of warnings, and I always recommend treating warnings as errors with that. Somehow it's coming up with a bounding box 200m tall, which is the main cause of the problem (the model is clearly not actually that size). I'm not sure whose bug this is; possibly ours, possibly three.js, but given the validator problems, I think you should file this on Unreal's glTF exporter. Let's see if we can get something sensible once the file's problems are fixed.
